sdrf_pipelines/maxquant/maxquant.py

```diff
@@ -2013,16 +2013,18 @@ def maxquant_convert(
             Variable_list.extend(j["mods"][1].split(","))
             fixedM_list = list(set(fixedM_list))
             Variable_list = list(set(Variable_list))
-            for F in fixedM_list:
-                string = doc.createElement("string")
-                string.appendChild(doc.createTextNode(F))
-                fixedModifications.appendChild(string)
-            for V in Variable_list:
-                if "Lys8" == V or "Lys6" == V or "Lys4" == V or "Arg10" == V or "Arg6" == V:
-                    continue
-                string = doc.createElement("string")
-                string.appendChild(doc.createTextNode(V))
-                variableModifications.appendChild(string)
+            if len(fixedM_list) > 0: 
+                for F in fixedM_list:
+                    string = doc.createElement("string")
+                    string.appendChild(doc.createTextNode(F))
+                    fixedModifications.appendChild(string)
+            if len(Variable_list) > 0:
+                for V in Variable_list:
+                    if "Lys8" == V or "Lys6" == V or "Lys4" == V or "Arg10" == V or "Arg6" == V:
```

Comment:
Here, you could use in instead like so: `if V in ["Lys8", "Lys6", ...]`

Reply:
Thanks, done. Was not my implementation anyways.